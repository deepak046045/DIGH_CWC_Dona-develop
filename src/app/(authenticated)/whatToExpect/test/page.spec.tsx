import { render } from '@testing-library/react';
import React from 'react';
import WhatToExpectPage from '../page';
import '@testing-library/jest-dom';

jest.mock('crypto', () => ({ randomUUID: () => '123456789' }));

describe('WhatToExpect Page', () => {
  it('should render page', () => {
    const { container } = render(<WhatToExpectPage />);
    expect(container).toMatchSnapshot();
  });
});
