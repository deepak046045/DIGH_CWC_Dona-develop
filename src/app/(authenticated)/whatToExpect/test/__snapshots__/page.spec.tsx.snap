// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhatToExpect Page should render page 1`] = `
<div>
  <div>
    What to expect page
  </div>
</div>
`;
